scripts/Waypoint.lua

```diff
@@ -1854,6 +1854,25 @@ function Course:addWaypointsForRows()
 	self:enrichWaypointData()
 end
 
+--- Gets waypoints without waypoints in between a row start and row end. 
+function Course:removeWaypointsBetweenRows()
```

Comment:
This works fine if turns are only at row ends. But we also have headland turns, so a better solution would be to mark the waypoints of a row during generation, as there we know exactly that this is a row, between to 180 turns.